app/clients/freshdesk.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+from flask import current_app
+
+
+class Freshdesk(object):
+    @staticmethod
+    def create_ticket(data):
+        ticket = {
+            'product_id': int(current_app.config['FRESH_DESK_PRODUCT_ID']),
+            'subject': data.get("support_type", "Support Request"),
+            'description': data["message"],
+            'email': data["email"],
+            'priority': 1,
+            'status': 2,
+            'tags': data.get("tags", []),
+        }
+
+        try:
+            response = requests.post(
+                f"{current_app.config['FRESH_DESK_API_URL']}/api/v2/tickets",
+                json=ticket,
+                auth=HTTPBasicAuth(current_app.config['FRESH_DESK_API_KEY'], "x")
+            )
+            response.raise_for_status()
+
+            return response.status_code
+        except requests.RequestException as e:
+            content = json.loads(response.content)
+            current_app.logger.warning(f"Failed to create Freshdesk ticket: {content['errors']}")
```

Comment:
I know we treated previous cases of users who couldn't send freshdesk tickets as a high priority, so I would ask if this log message is the only mean to know if there was a failure? If yes, we should consider it for an `error` level log.

Reply:
The line just after raises the `requests` exception 👌

Reply:
Yeah I didn't see it, that makes sense then. 🆗